CHANGELOG.rst

```diff
@@ -16,7 +16,12 @@ Unreleased
 
 *
 
-[1.2.0] - 2022-01-26
```

Comment:
This line should also be kept in the CHANGELOG since it is the header for previous release.